mrp_production_cancel_manager/views/mrp_production_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="mrp_production_form" model="ir.ui.view">
+        <field name="name">mrp.production.form</field>
+        <field name="model">mrp.production</field>
+        <field name="inherit_id" ref="mrp.mrp_production_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='action_cancel']" position="attributes">
```

Comment:
This code seems to be OK, but it might be incomplete, because there are already two Cancel buttons, and that xpath expression matches both:

https://github.com/odoo/odoo/blob/64a4ca83cd9b2f208c555942346bb62e31825266/addons/mrp/views/mrp_production_views.xml#L113-L117

Could you check if this code applies to both buttons, or only the first or last one?